From c834ddb8c5ca2957b7adb99a2a864f1892884e5e Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Thu, 30 Mar 2017 15:32:17 +0200
Subject: [PATCH 39/79] ARM64: dts: meson-gxbb-p20x: add analog amplifier node

Add the node for the analog sound amplifier Dioo 2125 found on the p20x
reference design

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>
Signed-off-by: Neil Armstrong <narmstrong@baylibre.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi b/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi
index 07b0f21..b9fc07c 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi
@@ -152,6 +152,12 @@
 		compatible = "everest,es7134";
 		status = "okay";
 	};
+
+	amp: analog-amplifier {
+		compatible = "dioo,dio2125";
+		enable-gpios = <&gpio GPIOH_3 0>;
+		status = "okay";
+	};
 };
 
 /* This UART is brought out to the DB9 connector */
-- 
1.9.1

